src/main/java/edu/greenblitz/robotName/commands/shooter/funnel/FunnelCommand.java

```diff
@@ -0,0 +1,20 @@
+package edu.greenblitz.robotName.commands.shooter.funnel;
+
+
+import edu.greenblitz.robotName.subsystems.shooter.Funnel.Funnel;
+import edu.greenblitz.robotName.utils.GBCommand;
+
+public abstract class FunnelCommand extends GBCommand {
+
+    protected Funnel funnel;
+
+    public FunnelCommand() {
+        funnel = Funnel.getInstance();
+        require(funnel);
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        funnel.stop();
+    }
```

Comment:
I wouldn't put here a stop, maybe you'll have a command that you don't want to stop the funnel at the end of it.